docs/docs/monitoring.md

```diff
@@ -86,6 +86,50 @@ When Prometheus is enabled in the status plugin (see [Configuration](./configura
 | last_success_bundle_request    | gauge       | Last successful bundle request in UNIX nanoseconds.    | STABLE |
 | bundle_loading_duration_ns     | histogram   | A histogram of duration for bundle loading.            | STABLE |
 
+## Metrics Overview
+
+OPA provides two ways to access performance metrics:
+
+1. **System-wide metrics** via the `/metrics` Prometheus endpoint - Aggregated metrics across all OPA operations
+2. **Per-query metrics** via API responses with `?metrics=true` - Metrics for individual query executions
+
+These serve different purposes: system metrics for monitoring and alerting, per-query metrics for debugging and optimization.
+
+
+## Accessing Metrics
+
+### System-Wide Metrics (Prometheus Endpoint)
+
+Access aggregated metrics across all OPA operations:
+
+- **URL**: `http://localhost:8181/metrics` (default configuration)
+- **Method**: HTTP GET
+- **Format**: Prometheus text format
+- **Contents**: All counters, timers, histograms, Go runtime metrics
+- **Use case**: Monitoring dashboards, alerting, performance trends
+
+### Per-Query Metrics
+
+Get metrics for individual policy evaluations:
```

Comment:
I don't really see this as 'monitoring' in the way the rest of this document is about monitoring.

Monitoring on this page is more about the instance as a whole, but these metrics are more about inspecting a given query. I think we can include a link to where these metrics are documented, but I am not sure that this is the right place to document this.